rooms/views.py

```diff
@@ -1,7 +1,70 @@
-from django.http  import JsonResponse
-from django.views import View
+from django.http       import JsonResponse
+from django.views      import View
+from datetime          import datetime
+
+from django.db.models  import Q
+
+from rooms.models      import Room, RoomAmenity, RoomHouseRule, RoomSchedule
+        
+class RoomListView(View):
+    def get(self, request):
+        category        = request.GET.get('category', 0)
+        check_in        = request.GET.get('check_in', "2022-01-01")
+        check_out       = request.GET.get('check_out', "2022-12-31")
+        guest           = request.GET.get('guest', 1)
+        price_min       = request.GET.get('price_min', 1)
+        price_max       = request.GET.get('price_max', 100000000)
+        instant_booking = request.GET.get('instant_booking', None)
+        bed             = request.GET.get('bed', 1)
+        bedroom         = request.GET.get('bedroom', 1)
+        bath            = request.GET.get('bath', 1)
+        amenity         = request.GET.getlist('amenity',None)
+            
+        page           = int(request.GET.get('offset', 0))
+        offset         = (page*20)
```

Comment:
20으로 정해지면, limit을 자유롭게 사용하지 못합니다.
```suggestion
        offset         = (page*limit)
```

Reply:
확인했습니다